engines/touche/touche.h

```diff
@@ -36,12 +36,12 @@
 #include "touche/console.h"
 
 /**
- * This is the namespace of the Touche engine.
+ * This is the namespace of the Touché engine.
```

Comment:
This references the engine, not the game title

Reply:
Changed to `Touche`. It does seem that other classes do use the uppercase version of the engine id in this case. E.g. `toon.h` says `This is the namespace of the Toon engine.`